NuGet.config

```diff
@@ -15,6 +15,7 @@
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
     <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
     <add key="vssdk@Local" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk%40Local/nuget/v3/index.json" />
```

Comment:
There is already `vssdk@Local` feed above where you can pull Interop package, no need to the new vssdk feed.

Reply:
Done, thanks !